app-server/app/src/main/scala/organizations/Organizations.scala

```diff
@@ -0,0 +1,328 @@
+package com.azavea.rf.organization
+
+import scala.concurrent.{Future, ExecutionContext}
+import scala.util.{Success, Failure, Try}
+import slick.lifted.Query
+import java.sql.Timestamp
+import org.postgresql.util.PSQLException
+
+import com.lonelyplanet.akka.http.extensions.{PageRequest, Order}
+
+import com.azavea.rf.datamodel.latest.schema.tables.{
+  Users,
+  Organizations,
+  OrganizationsRow,
+  UsersToOrganizations,
+  UsersToOrganizationsRow
+}
+
+import com.azavea.rf.utils.{
+  Database,
+  PaginatedResponse,
+  UserErrorException
+}
+import com.azavea.rf.user.UserRoles
+
+
+case class UserWithRole(id: String, role: String, createdAt: Timestamp, modifiedAt: Timestamp)
+
+
+case class UserWithRoleCreate(id: String, role: String) {
+  def toUserWithRole(): UserWithRole = {
+    val now = new Timestamp((new java.util.Date()).getTime())
+    UserWithRole(id, role, now, now)
+  }
+}
+
+
+case class OrganizationsRowCreate(name: String) {
+  def toOrganizationsRow(): OrganizationsRow = {
+    val id = java.util.UUID.randomUUID()
+    val now = new Timestamp((new java.util.Date()).getTime())
+    OrganizationsRow(id, now, now, name)
+  }
+}
+
+
+object OrganizationService {
+  def applyOrgSort(
+    query: Query[Organizations, Organizations#TableElementType, Seq],
+    sortMap: Map[String, Order]
+  )(
+    implicit
```

Comment:
I find breaking up the implicit onto a separate line a bit hard to parse. I think something along the lines of what's in `User.scala` would look a bit better. Something like:
```
def applyOrgSort(
  query: Query[Organizations, Organizations#TableElementType, Seq],
  sortMap: Map[String, Order]
)(implicit database: Database, ec: ExecutionContext): 
    Query[Organizations, Organizations#TableElementType, Seq] = {
```